src/loaders/lottie/tvgLottieInterpolator.cpp

```diff
@@ -40,20 +40,20 @@
 #define SUBDIVISION_MAX_ITERATIONS 10
 
 
-static inline float _A(float aA1, float aA2) { return 1.0f - 3.0f * aA2 + 3.0f * aA1; }
-static inline float _B(float aA1, float aA2) { return 3.0f * aA2 - 6.0f * aA1; }
-static inline float _C(float aA1) { return 3.0f * aA1; }
+static inline float tvg_A(float aA1, float aA2) { return 1.0f - 3.0f * aA2 + 3.0f * aA1; }
```

Comment:
The issue likely arises from the pre-processor converting A, B, and C into numeric values. I don't know about the compiler much, assuming the current name is too short for it.

Here is the static function name style guideline:
```
static float _funcName(..);

//so like:
_A(...) -> _constA(...)     //the macro is to determine the constant value of the equation.
_B(...) -> _constB(...)
_C(...) -> _constC(...)
```